testutil/integration/network/setup.go

```diff
@@ -0,0 +1,209 @@
+package network
+
+import (
+	"time"
+
+	"github.com/evmos/evmos/v14/app"
+	"github.com/evmos/evmos/v14/encoding"
+
+	"cosmossdk.io/simapp"
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/testutil/mock"
+
+	sdkmath "cosmossdk.io/math"
+	dbm "github.com/cometbft/cometbft-db"
+	"github.com/cometbft/cometbft/libs/log"
+	tmtypes "github.com/cometbft/cometbft/types"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
+	simutils "github.com/cosmos/cosmos-sdk/testutil/sims"
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	epochstypes "github.com/evmos/evmos/v14/x/epochs/types"
+	infltypes "github.com/evmos/evmos/v14/x/inflation/types"
+)
+
+// createValidatorSet creates validator set with the amount of validators specified
+// with the default power of 1.
+func createValidatorSet(numberOfValidators int) *tmtypes.ValidatorSet {
+	// Create validator set
+	tmValidators := make([]*tmtypes.Validator, numberOfValidators)
+	for i := 0; i < numberOfValidators; i++ {
+		privVal := mock.NewPV()
+		pubKey, _ := privVal.GetPubKey()
+		validator := tmtypes.NewValidator(pubKey, 1)
+		tmValidators[i] = validator
+	}
+
+	return tmtypes.NewValidatorSet(tmValidators)
+}
+
+// fundAndCreateGenesisAccounts funds accounts
+// create genesis accounts
+func createGenesisAccounts(accounts []sdktypes.AccAddress) []authtypes.GenesisAccount {
+	numberOfAccounts := len(accounts)
+	genAccounts := make([]authtypes.GenesisAccount, numberOfAccounts)
+	for i, acc := range accounts {
+		baseAcc := authtypes.NewBaseAccount(acc, nil, 0, 0)
+		genAccounts[i] = baseAcc
+	}
+	return genAccounts
+}
+
+func createBalances(accounts []sdktypes.AccAddress, coin sdktypes.Coin) []banktypes.Balance {
+	numberOfAccounts := len(accounts)
+	fundedAccountBalances := make([]banktypes.Balance, numberOfAccounts)
+	for i, acc := range accounts {
+		balance := banktypes.Balance{
+			Address: acc.String(),
+			Coins:   sdktypes.NewCoins(coin),
+		}
+
+		fundedAccountBalances[i] = balance
+	}
+	return fundedAccountBalances
+}
+
+func createEvmosApp(chainID string) *app.Evmos {
```

Comment:
nit: maybe include `loadLatest` as a function param here

Reply:
I think its a good but to keep things as simple as possible do you think its ok to keep it like this until we find a situation where its needed?